src/Wallet.js

```diff
@@ -41,6 +41,40 @@ exports._submitTxNami = txHex => nami => () => {
       });
 };
 
+// _enableGero :: Effect (Promise NamiConnection)
+exports._enableGero = () => window.cardano.gero.enable();
+
+// _getGeroAddress :: NamiConnection -> Effect (Promise String)
```

Comment:
Here as well, and for all of the other type signatures in this module

Reply:
Fixed